fms_fsdp/utils/dataloader_utils.py

```diff
@@ -56,21 +68,13 @@ def get_data_loader(cfg, rank, world_size):
         Rank of current distributed worker. Used for handling dataset sharding logic.
     world_size : int
         Number of distributed workers. Used for handling dataset sharding logic.
+    postprocess : List[Callable]
```

Comment:
can we include input/output types here

Reply:
Sorry, I didn't follow

Reply:
parameterize the callable (`List[Callable[[Parameter1Type, Parameter2Type], ReturnType]]`) for whatever the parameters/return are

Reply:
@daviswer

Reply:
Maybe an example would help if that can be added here in the docs